node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/v2/Authorizer.java

```diff
@@ -74,6 +87,11 @@ private boolean isNodeType(NodeType type, Principal principal) {
                                            .orElse(false);
     }
 
+    /** Returns whether given principal is the local host (this node) */
+    private boolean isHost(Principal principal) {
```

Comment:
can we use another name so that this is not mistaken to be a docker host? isLocalHost or something similar

Reply:
Done.